specification/cognitiveservices/data-plane/AzureOpenAI/authoring/preview/2023-04-01-preview/images.json

```diff
@@ -0,0 +1,348 @@
+      },
+      "InnerError": {
+         "type": "object",
+         "description": "The inner error if a task in a batch failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Operation failed as a result of our safety system."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "ContentFilter",
+                  "Internal",
+                  "BlockedImages"
+               ],
+               "description": "The error code."
+            }
+         }
+      },
+      "Error": {
+         "type": "object",
+         "description": "The error if the request or operation failed.",
+         "properties": {
+            "message": {
+               "type": "string",
+               "description": "The error message.",
+               "example": "Required query parameter 'api-version' is missing."
+            },
+            "code": {
+               "type": "string",
+               "enum": [
+                  "BadRequest",
+                  "ContentFilter",
+                  "Internal",
+                  "TooManyRequests",
+                  "Unauthorized",
+                  "NotFound",
+                  "OperationNotInTerminalState",
+                  "BlockedImages",
+                  "RequestTimout"
+               ],
+               "description": "The error code."
+            },
+            "details": {
+               "type": "array",
+               "items": {
+                  "$ref": "#/definitions/InnerError"
+               }
+            }
+         },
+         "required": [
+            "message",
+            "code"
+         ]
+      },
+      "ErrorResponse": {
+         "type": "object",
+         "properties": {
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         },
+         "required": [
+            "error"
+         ]
+      },
+      "ImageResult": {
+         "type": "object",
+         "description": "The image url if successful, and an error otherwise.",
+         "properties": {
+            "url": {
+               "type": "string",
+               "description": "The image url.",
+               "example": "https://www.contoso.com"
+            },
+            "error": {
+               "$ref": "#/definitions/Error"
+            }
+         }
+      },
+      "OperationResult": {
+         "type": "array",
+         "description": "The result data of the operation, if successful",
+         "items": {
+            "$ref": "#/definitions/ImageResult"
+         }
+      },
+      "OperationResponse": {
+         "type": "object",
+         "properties": {
+            "id": {
+               "type": "string",
+               "description": "The ID of the operation.",
+               "example": "59c7e2ea-7450-4e85-9762-55c78727f4c7"
+            },
+            "created": {
```

Comment:
>created [](http://example.com/codeflow?start=13&length=7)

created_at? to be consistent with expires(other openai properties use _at)

Reply:
"created" comes from OpenAPI, so be consistent I tihnk we should keep this name?

Reply:
As discussed, we will use "created"